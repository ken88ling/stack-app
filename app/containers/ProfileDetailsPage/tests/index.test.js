import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import ProfileDetails from '../index';

describe('<FeaturePage />', () => {
  it('should render its heading', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <ProfileDetails />
      </IntlProvider>,
    );

    expect(firstChild).toMatchSnapshot();
  });
});
