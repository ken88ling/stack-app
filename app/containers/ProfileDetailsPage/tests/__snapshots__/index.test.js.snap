// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FeaturePage /> should render its heading 1`] = `
<div>
  <h1
    class="H1-tncdg6-0 hpAKSC"
  >
    <span>
      Features
    </span>
  </h1>
</div>
`;
